src/test/regress/multi_1_schedule

```diff
@@ -324,11 +323,9 @@ test: ssl_by_default
 # object distribution tests
 # ---------
 test: distributed_types distributed_types_conflict disable_object_propagation distributed_types_xact_add_enum_value
-test: check_mx
 test: distributed_functions distributed_functions_conflict
 test: distributed_collations distributed_collations_conflict
 test: distributed_procedure
```

Comment:
The last line of `distributed_procedure` is :
```SQL
SELECT stop_metadata_sync_to_node('localhost', :worker_1_port);
SELECT stop_metadata_sync_to_node('localhost', :worker_2_port);
```

I think we should remove that?